docs/build_version_doc/build_all_version.sh

```diff
@@ -122,6 +122,8 @@ function checkout () {
     git pull
     # master gets warnings as errors for Sphinx builds
     OPTS="-W"
+  else
```

Comment:
OPTS is already empty (set at line 47). Why is this required? If tag is master OPTS is set to "-w" otherwise it is left as is which is empty.

Reply:
Once master gets called OPTS is set, but when another version is called it should be unset.